templates/rhdm74-trial-ephemeral.yaml

```diff
@@ -696,7 +744,7 @@ objects:
             value: "${DECISION_CENTRAL_MAVEN_USERNAME}"
           - name: RHDMCENTR_MAVEN_REPO_PASSWORD
             value: "${DEFAULT_PASSWORD}"
-          - name: EXTERNAL_MAVEN_REPO_ID
+          - name: EXTERNAL_MAVEN_REPO_ID 
```

Comment:
@rhtevan Please make sure there are no trailing spaces in the files.

Reply:
@errantepiphany fixed.